pandas/tests/indexing/test_iloc.py

```diff
@@ -767,3 +767,11 @@ def test_iloc_setitem_series_duplicate_columns(self):
         )
         df.iloc[:, 0] = df.iloc[:, 0].astype(np.float64)
         assert df.dtypes.iloc[2] == np.int64
+
+
+def test_iloc_setitem_bool_array():
```

Comment:
could also be put somewhere in pandas\tests\frame\indexing\

Reply:
Yeah, that's the file I looked for there :) Thanks